DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,374 @@
+    export CLUSTER_NODE_SIZE="s-2vcpu-4gb"        # change as required
+    export CLUSTER_NODE_COUNT=2                   # need 2 nodes at least to test the scenarios
+    export CLUSTER_NODE_POOL_NAME="mbt-np"        # change as required
+    export CLUSTER_NODE_POOL_TAG="mbt-cluster"    # change as required
+    export CLUSTER_NODE_POOL_LABEL="type=basic"   # change as required
+
+    doctl k8s cluster create "$CLUSTER_NAME" \
+      --auto-upgrade=false \
+      --node-pool "name=${CLUSTER_NODE_POOL_NAME};size=${CLUSTER_NODE_SIZE};count=${CLUSTER_NODE_COUNT};tag=${CLUSTER_NODE_POOL_TAG};label=${CLUSTER_NODE_POOL_LABEL}" \
+      --region "$CLUSTER_REGION"
+    ```
+
+    The output looks similar to the following:
+    ```
+    Notice: Cluster is provisioning, waiting for cluster to be running
+    ......................................................................
+    Notice: Cluster created, fetching credentials
+    Notice: Adding cluster credentials to kubeconfig file found in "~/.kube/config"
+    Notice: Setting current-context to do-lon1-mobius-testing-cluster
+    ID                                      Name                       Region    Version        Auto Upgrade    Status     Node Pools
+    11bdd0f1-8bd0-42dc-a3af-7a83bc319295    mobius-testing-cluster     lon1      1.21.2-do.2    false           running    basicnp
+    ```
+
+6. Get worker nodes status:
+
+    ```bash
+    kubectl get nodes
+    ```
+
+    The output looks similar to the following:
+    ```
+    NAME            STATUS   ROLES    AGE     VERSION
+    basicnp-8k4ep   Ready    <none>   2m52s   v1.21.2
+    basicnp-8k4es   Ready    <none>   2m16s   v1.21.2
+    ```
+    ### Note:
+
+      If for some reason the `kubectl` context was not set properly we can always fix it via:
+      
+      ```bash
+      doctl k8s cluster kubeconfig save "$CLUSTER_NAME"
+      ```
+
+## Digital Mobius
+
+### Overview
+
+[Digital Mobius](https://github.com/Qovery/digital-mobius) is an open source project written in the `GO` language which can automatically recycle unhealthy nodes from a `DigitalOcean Kubernetes` cluster (DOKS).
+
+### How it Works
+
+The logic inside the application is watching for nodes that are in an unhealthy state at a regular interval specified by the user. A node is considered to be unhealthy if the [Node Condition](https://kubernetes.io/docs/concepts/architecture/nodes/#condition) is set to `Ready` and the status to `False` or `Unknown`. `Digital Mobius` provides also a user configurable flag that specifies how much time has to pass before an unhealthy node needs to be recycled. If all the conditions enumerated previously are met then the affected node(s) will be re-created. It does so via a call to the DigitalOcean's Kubernetes [Delete Kubernetes Node](https://docs.digitalocean.com/reference/api/api-reference/#operation/delete_kubernetes_node) `REST API`.
```

Comment:
>A node is considered to be unhealthy if the `Node Condition`is set to `Ready` and the status to `False` or `Unknown`.

Looking [at the Digital Mobius code](https://github.com/Qovery/digital-mobius/blob/32179d0ffde6016a51b2b3b3b95dcd187e19fbed/utils/k8s.go#L101), it seems like it only considers nodes unhealthy if the status is `Unknown` (plus a few other conditions applying).

Reply:
That's true. I just forgot to be more explicit. I updated the flow diagram as well to better reflect all the conditions, hopefully. Thanks.